diff --git a/node_modules/@snowpack/app-scripts-react/jest/babelTransform.js b/node_modules/@snowpack/app-scripts-react/jest/babelTransform.js
index e0bbaa2..bdd7600 100644
--- a/node_modules/@snowpack/app-scripts-react/jest/babelTransform.js
+++ b/node_modules/@snowpack/app-scripts-react/jest/babelTransform.js
@@ -11,6 +11,8 @@ const babelJest = require('babel-jest');
 const importMetaBabelPlugin = require('./importMetaBabelPlugin');
 
 module.exports = babelJest.createTransformer({
-  presets: ['babel-preset-react-app', '@babel/preset-react', '@babel/preset-typescript'],
+  presets: ['babel-preset-react-app', ['@babel/preset-react', {
+    runtime: "automatic"
+  }], '@babel/preset-typescript'],
   plugins: [[importMetaBabelPlugin]],
 });
