diff --git a/node_modules/snowpack/lib/cjs/build/optimize.js b/node_modules/snowpack/lib/cjs/build/optimize.js
index 156b0ab..91efe78 100644
--- a/node_modules/snowpack/lib/cjs/build/optimize.js
+++ b/node_modules/snowpack/lib/cjs/build/optimize.js
@@ -351,7 +351,7 @@ async function runEsbuildOnBuildDirectory(bundleEntrypoints, allFiles, config) {
         minify: config.optimize.minify,
         target: config.optimize.target,
         external: Array.from(new Set(allFiles.map((f) => '*' + path_1.default.extname(f))))
-            .filter((ext) => ext !== '*.js' && ext !== '*.mjs' && ext !== '*.css' && ext !== '*')
+            .filter((ext) => !["*.js", "*.mjs", "*.css", "*", "*.png", "*.gif", "*.jpg", "*.webp", "*.jpeg"].includes(ext))
             .concat((_b = (_a = config.packageOptions) === null || _a === void 0 ? void 0 : _a.external) !== null && _b !== void 0 ? _b : []),
         charset: 'utf8',
     });
